Steamworks/src/org/usfirst/frc/team4786/robot/Robot.java

```diff
@@ -32,9 +33,8 @@
 	@Override
 	public void robotInit() {
 		oi = new OI();
-//		chooser.addDefault("Default Auto", new ExampleCommand());
+		chooser.addObject("Test PID", new DriveToPosition(1));
```

Comment:
This is fine now, but we will need to set it to display something other than "Test PID" later.